src/mumble/ConnectDialog.cpp

```diff
@@ -1406,13 +1406,8 @@ void ConnectDialog::onUpdateLanList(const QList< BonjourRecord > &list) {
 	}
 }
 
-void ConnectDialog::onLanBrowseError(DNSServiceErrorType err) {
-	qWarning() << "Bonjour reported browser error " << err;
-}
-
-void ConnectDialog::onLanResolveError(BonjourRecord br, DNSServiceErrorType err) {
-	qlBonjourActive.removeAll(br);
-	qWarning() << "Bonjour reported resolver error " << err;
+void ConnectDialog::onLanResolveError(const BonjourRecord record) {
+	qlBonjourActive.removeAll(record);
```

Comment:
Is `qlBonjourActive` even used anymore? And if it is: Does it actually still refer to Bonjour-stuff?

Reply:
It does, due to it containing `BonjourRecord` elements.